docs/source/_templates/page.html

```diff
@@ -0,0 +1,116 @@
+of this software and associated documentation files (the "Software"), to
+deal in the Software without restriction, including without limitation the
+rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+sell copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+-->
+
+{%- extends "!page.html" %}
+
+{% block footer %}
+<div class="related-pages">
+    {% if next -%}
+    <a class="next-page" href="{{ next.link }}">
+        <div class="page-info">
+        <div class="context">
+            <span>{{ _("Next") }}</span>
+        </div>
+        <div class="title">{{ next.title }}</div>
+        </div>
+        <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
+    </a>
+    {%- endif %}
+    {% if prev -%}
+    <a class="prev-page" href="{{ prev.link }}">
+        <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
+        <div class="page-info">
+        <div class="context">
+            <span>{{ _("Previous") }}</span>
+        </div>
+        {% if prev.link == pathto(master_doc) %}
+        <div class="title">{{ _("Home") }}</div>
+        {% else %}
+        <div class="title">{{ prev.title }}</div>
+        {% endif %}
+        </div>
+    </a>
+    {%- endif %}
+</div>
+<div class="bottom-of-page">
+    <div class="left-details">
+    {%- if show_copyright %}
+    <div class="copyright">
+        {%- if hasdoc('copyright') %}
+        {% trans path=pathto('copyright'), copyright=copyright|e -%}
+            <a href="{{ path }}">Copyright</a> &#169; {{ copyright }}
+        {%- endtrans %}
+        {%- else %}
+        {% trans copyright=copyright|e -%}
+            Copyright &#169; {{ copyright }}
+        {%- endtrans %}
+        {%- endif %}
+    </div>
+    {%- endif %}
+    {% trans %}Made with {% endtrans -%}
+    {%- if show_sphinx -%}
+    {% trans %}<a href="https://www.sphinx-doc.org/">Sphinx</a> and {% endtrans -%}
+    <a class="muted-link" href="https://pradyunsg.me">@pradyunsg</a>'s
+    {% endif -%}
+    {% trans %}
+    <a href="https://github.com/pradyunsg/furo">Furo</a>
+    {% endtrans %}
+    {%- if last_updated -%}
+    <div class="last-updated">
+        {% trans last_updated=last_updated|e -%}
+        Last updated on {{ last_updated }}
+        {%- endtrans -%}
+    </div>
+    {%- endif %}
+    <div class="legal-info">
+        <script type="text/javascript" src="{{ pathto('_static/shortbreadv1.js',1 ) }}"></script>
+        <script type="text/javascript">
+                const shortbread = AWSCShortbread({
+                    domain: ".amazonaws.com",
+    });
+                shortbread.checkForCookieConsent();
+        </script>
+        <a href="http://aws.amazon.com/privacy">Privacy</a> | <a href="http://aws.amazon.com/terms">Site Terms</a> | <a
+            href="#" onclick="shortbread.customizeCookies();">Cookie preferences</a>
+    </div>
```

Comment:
This is the only change from the `{% block footer %}` in the original furo page.html template.